PWGHF/HFC/TableProducer/correlatorDstarHadrons.cxx

```diff
@@ -218,8 +237,28 @@ struct HfCorrelatorDstarHadrons {
                               assocParticle.pt(),
                               timestamp,
                               binNumber);
+          if (enableSeparateTables == true) {
+            rowsDstar(collision.globalIndex(),
+                      gItriggerParticle,
+                      triggerParticle.phi(),
+                      triggerParticle.eta(),
+                      triggerParticle.pt(),
+                      triggerParticle.invMassAntiDstar(),
+                      triggerParticle.invMassD0Bar(),
+                      timestamp,
+                      binNumber);
+          }
```

Comment:
You can avoid filling `rowsDstar` at two places. The two instances just differ in the two masses, which you can assign inside this `if else` block and then have a single call of `rowsDstar()` outside.

Reply:
Same for `rowsDstarHadronPair`.